docker/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgresql
+  pool: 5
+  timeout: 5000
+  username: postgres
+  password:
+  host: db
+
+development:
+  <<: *default
+  database: timesheet-dev
+
+test:
+  <<: *default
+  database: timesheet_test
+
+staging: &staging
```

Comment:
I don't imagine that the `staging` or `production` environments are actually used inside of Docker (at least not yet). The way things are presently configured, I don't think `test` is either. For clarity's sake, would you mind removing unused sections?